### ISSUE TITLE
Is there any way to determine the size of the repo without cloning it?

### ISSUE THREAD

**Issue by PlanetGamingGG** [czproject/git-php#30]
Hi, I was planning to hook this up with my other scripts but I wanted to limit the maximum possible size my script will handle.
How can we determine the remote repo size before cloning it?
Thanks.

**Comment by janpecha** [comment]
Hello, currently it isn't possible and I think Git doesn't support it.

**Comment by janpecha** [comment]
Yes, GitHub API can be solution but it's out of scope of this project. Maybe some library for GitHub API can help.
